omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -285,8 +285,8 @@
 var_base = '/var/opt/opscode'
 log_base = '/var/log/opscode'
 
-default['private_chef']['elasticsearch']['enable'] = false
-default['private_chef']['elasticsearch']['first_internal_install'] = false
+default['private_chef']['elasticsearch']['enable'] = true
```

Comment:
We need to make a change to libraries/private_chef.rb to disable elasticsearch on the frontend.